bluebottle/bb_accounts/views.py

```diff
@@ -162,7 +162,9 @@ def create(self, request, *args, **kwargs):
             self.request.user.last_logout = timezone.now()
             self.request.user.save()
 
-        return response.Response('', status=status.HTTP_204_NO_CONTENT)
+            return response.Response('', status=status.HTTP_204_NO_CONTENT)
+
+        return response.Response('', status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
This works, but it is probably a little clear to just add a correct permission class:

```
permission_classes = (IsAuthenticated, )
```

Reply:
Ok done. looks much better.